src/Tribe/Editor.php

```diff
@@ -84,12 +131,12 @@ public function deactivate_blocks_editor_organizer( $is_enabled, $post_type ) {
 	/**
 	 * For now we don't use Blocks editor on the Post Type for Venues
 	 *
-	 * @todo  see https://core.trac.wordpress.org/ticket/45275
+	 * @todo   see https://core.trac.wordpress.org/ticket/45275
 	 *
 	 * @since  4.7
 	 *
-	 * @param  boolean $is_enabled
-	 * @param  string  $post_type
+	 * @param boolean $is_enabled
```

Comment:
The param name is `bool` by convention.
```suggestion
	 * @param bool $is_enabled
```